library/src/main/java/com/chuckerteam/chucker/internal/support/ChuckerFileProvider.kt

```diff
@@ -6,4 +6,4 @@ import androidx.core.content.FileProvider
  * We need our own subclass so we don't conflict with other [FileProvider]s
  * See: https://github.com/ChuckerTeam/chucker/issues/409
  */
-class ChuckerFileProvider : FileProvider()
+internal class ChuckerFileProvider : FileProvider()
```

Comment:
Can you verify that this is not crashing at runtime?
This class is referenced inside the AndroidManifest and I'm unsure if `internal` visibility is valid here.

Reply:
I was worrying about it as well. However, I had no issues on API 19 and 30 when tested file sharing in our sample app.

Reply:
It should behave the same. `internal` compiles down to `public` so in the byte code it is the same thing unless I'm missing something.

Reply:
> internal compiles down to public so in the byte code it is the same thing unless I'm missing something.

It should actually mangle the name of the class to be `ChuckerFileProvider$com_github_ChuckerTeam_Chucker_library` if I'm not mistake (at least that's [what happens for method names](https://ncorti.com/blog/name-mangling-in-kotlin))

Reply:
From my quick check, it should be ok. 

```kotlin
internal class Foo {
    internal fun foo() = Unit
}

internal fun foo() = Unit
```

```sh
kotlinc Foo.kt
javap Foo.class
```

```java
Compiled from "Foo.kt"
public final class Foo {
  public final void foo$main();
  public Foo();
}
```

Reply:
TIL 👍  Then it's good to go for me